testserver/testserver_test.go

```diff
@@ -32,22 +32,44 @@ func TestRunServer(t *testing.T) {
 			instantiation: func(t *testing.T) (*sql.DB, func()) { return testserver.NewDBForTest(t) },
 		},
 		{
-			name:          "Secure",
+			name:          "SecureClientCert",
 			instantiation: func(t *testing.T) (*sql.DB, func()) { return testserver.NewDBForTest(t, testserver.SecureOpt()) },
 		},
+		{
+			name: "SecurePassword",
+			instantiation: func(t *testing.T) (*sql.DB, func()) {
+				return testserver.NewDBForTest(t, testserver.SecureOpt(), testserver.RootPasswordOpt("asd1828585!@$"))
```

Comment:
not sure if we need different passwords for all these. we could just make a local password variable to reduce the number of magic strings and make the function call self-documenting. no strong preference here, just a thought if you'd like to do that.